config/production/config.yaml

```diff
@@ -0,0 +1,4 @@
+aws_infra_templates_root_url: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra
+default_stack_tags:
+  Department: IBC
```

Comment:
You can include `Project: Infrastructure` and `OwnerEmail: dpe@sagebase.org`.

Reply:
added in latest commit